### PR TITLE
SDGF ghost poll message made terse, other tweaks.

### PR DESCRIPTION
**Pull request by Ghommie**
## About The Pull Request
Making SDGF ghost poll message less verbose, more terse. Also making the experimental cloner's in line with the former. Implementing a ghost poll ignore option for both. Also a operator typo fix.

## Why It's Good For The Game
Dang, the SDGF ghost poll message both discourages ghosts from optin in and annoys them with its overly long message that doesn't remotely fit in the pop up box.

## Changelog
:cl:
tweak: Made SDGF ghost poll message less verbose, made the experimental cloner's complaint with the former, and added ghost poll ignore options for both. 
/:cl:

### CODE REVIEW

**Comment by Poojawa** [comment]
can this be applied to the stupid experimental cloner prompt as well

**Comment by Ghommie** [comment]
> 
> 
> can this be applied to the stupid experimental cloner prompt as well

Already did.

File: modular_citadel/code/modules/reagents/chemistry/reagents/SDGF.dm [hunk 1]
```diff
@@ -68,11 +68,10 @@ IMPORTANT FACTORS TO CONSIDER WHILE BALANCING
 			startHunger = M.nutrition
 			if(pollStarted == FALSE)
 				pollStarted = TRUE
-				candies = pollGhostCandidates("Do you want to play as a clone of [M], and do you agree to respect their character and act in a similar manner to them? Do not engage in ERP as them unless you have LOOC permission from them, and ensure it is permission from the original, not a clone.")
+				candies = pollGhostCandidates("Do you want and agree to play as a clone of [M], respect their character and not engage in ERP without permission from the original?", ignore_category = POLL_IGNORE_CLONE)
 				log_game("FERMICHEM: [M] ckey: [M.key] has taken SDGF, and ghosts have been polled.")
 		if(20 to INFINITY)
 			if(LAZYLEN(candies) && playerClone == FALSE) //If there's candidates, clone the person and put them in there!
-				log_game("FERMICHEM: [M] ckey: [M.key] is creating a clone, controlled by [candies]")
```
Comment (Ghommie) [inline modular_citadel/code/modules/reagents/chemistry/reagents/SDGF.dm:75]: Useless log_game, [candies] not even a key, that's why I removed it.
Reagents/chemistry logs should be ported someday, gosh.

Comment (Poojawa) [reply modular_citadel/code/modules/reagents/chemistry/reagents/SDGF.dm:75]: or just be tagged under testing

**Review by deathride58 — APPROVED** [review]

**Pull request merged**